tests/integration/test_modified_config_files_validation.py

```
@@ -0,0 +1,392 @@
+        """Get path to PR agent config."""
+        return Path(__file__).parent.parent.parent / ".github" / "pr-agent-config.yml"
+    
+    @pytest.fixture
+    def config_data(self, config_path: Path) -> Dict[str, Any]:
+        """
+        Load and parse the PR Agent YAML configuration file.
+        
+        Returns:
+            config (Dict[str, Any]): Mapping representing the parsed YAML configuration.
+        """
+        with open(config_path, 'r') as f:
```

Ruff (0.14.8)

Unnecessary mode argument

Remove mode argument

(UP015)